staging/src/k8s.io/controller-manager/service/config/doc.go

```diff
@@ -1,7 +1,5 @@
-// +build !ignore_autogenerated
```

Comment:
This move seems wrong. I think we'll end up with the wrong file history.

Reply:
Thanks. Updated to remove this file history